docs/FAQ.md

```diff
@@ -0,0 +1,83 @@
+# Registry past-and-present cheat sheet
+
+> Everything You Always Wanted to Know about the Registry
+
+## What's a Registry?
+
+It's a server-side storage and delivery software that let you publish and exchange docker images.
+
+Whenever you do a `docker pull` or `docker push`, your engine communicates with a registry.
+
+## So, does the Docker Hub contain a registry?
+
+Yes.
+
+That "Hub registry" is operated by Docker.
+
+It is free to use for public content, and offers paid-for private storage as well.
+
+Support for the hub is handled directly by Docker, over here: https://support.docker.com
+
```

Comment:
A mention what the differences are between Docker Hub and the open source project? (Web interface not included, etc)